src/conduit/components/forms.cljs

```diff
@@ -62,6 +66,28 @@
                                                      :email email
                                                      :password password})))})
 
+(def settings-form
+  {:fields {:image {:placeholder "URL of profile picture"}
+            :username {:placeholder "Username"}
+            :bio {:placeholder "Short bio about you" :field-type :textarea}
+            :email {:placeholder "Email"}
+            :password {:placeholder "New Password" :type "password"}}
+   :validators {:username [[#(not (empty? %)) "Please enter username"]]
+                :email [[#(not (empty? %)) "Please enter email"]
+                        [form-helper/email? "Invalid Email"]]}
+   :on-init
```

Comment:
What is `:on-init`? Why do we need it here?

Reply:
`:on-init` is used for populating form data with stored values. In this particular case, it loads the logged-in user profile into form data.